content/applications/sales/crm/optimize/partner_autocomplete.rst

```diff
@@ -1,48 +1,73 @@
-===================================================
-Enrich your contacts base with Partner Autocomplete
-===================================================
+==============================================
+Enrich contacts base with partner autocomplete
+==============================================
 
-Parter Autocomplete helps you to enrich your contacts database with corporate data. Select one of
-the companies suggested in the dropdown, and quickly get all the information you need.
+*Partner autocomplete* helps to enrich the contacts database with corporate data. Select one of the
+companies suggested in the drop-down menu, and quickly get all the information.
```

Comment:
could you add more specificity when it comes to the what drop-down menu, and where they could find such a thing in the database?

Reply:
UPDATE: I understand this can happen throughout any module in the db that has a 'Company' field, but it might be helpful to add that information right here at the top, so as to avoid any confusion with the readers.